src/Messages/Interactions/TSInteraction.m

```diff
@@ -72,11 +72,6 @@ - (uint64_t)millisecondsTimestamp {
     return self.timestamp;
 }
 
-- (NSDate *)date {
-    uint64_t seconds = self.timestamp / 1000;
```

Comment:
Hmm, this problem exists elsewhere too.  I'm pushing another commit that fixes it.